### PR TITLE
AddPythonProject sample

### PR DESCRIPTION
**Pull request by mitchdenny**
Builds on #362 but adds Python. Not included in the main PR because I don't want to block that if we have dependency issues with Python.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -431,3 +431,2037 @@ FodyWeavers.xsd
 **/*/azure.yaml
 **/*/next-steps.md
 
+samples/AspireWithPython/InstrumentedPythonProject/.venv/pyvenv.cfg
```
Comment (IEvangelist) [inline .gitignore:434]: Yuck, just ignore `**/*/.venv/` and be done with it. No reason individually ignore all these files.

Comment (mitchdenny) [reply .gitignore:434]: OMG just right clicked on the folder and selected ignore.

File: samples/AspireWithPython/AspireWithPython.AppHost/Program.cs [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿var builder = DistributedApplication.CreateBuilder(args);
+
+builder.AddPythonProject("instrumented-python-project", "..\\InstrumentedPythonProject", "main.py")
```
Comment (IEvangelist) [inline samples/AspireWithPython/AspireWithPython.AppHost/Program.cs:3]: NIT: .NET supports both backslash and forward slash, I prefer to standardize on forward slash as it's less characters to type and it doesn't require an escape sequence.

```suggestion
builder.AddPythonProject("instrumented-python-project", "../InstrumentedPythonProject", "main.py")
```

**Review by IEvangelist — APPROVED** [review]

**Comment by mitchdenny** [comment]
Merging on red as build failures seem unrelated.

**Pull request merged**